lib/Target/RISCV/MCTargetDesc/RISCVMCTargetDesc.cpp

```diff
@@ -59,6 +60,57 @@ static MCInstPrinter *createRISCVMCInstPrinter(const Triple &T,
   return new RISCVInstPrinter(MAI, MII, MRI);
 }
 
+/// Select the RISCV feature for the given triple.
+StringRef Feature::parseRISCVTriple(const Triple &TT, StringRef CPU) {
+  StringRef MArch = TT.getArchName();
+  StringRef Ext = MArch.substr(7);
+  std::string RISCVFeature;
+
+  if (MArch.startswith("riscv64")) {
+    RISCVFeature = "+64bit";
+  }
+
+  for (auto ext : Ext) {
```

Comment:
`char c` would be clearer rather than using auto.

Reply:
Using char instead of auto here.